nautobot/dcim/tests/test_views.py

```diff
@@ -1280,8 +1640,20 @@ def setUpTestData(cls):
             "feed_leg": PowerOutletFeedLegChoices.FEED_LEG_B,
         }
 
+    def _edit_object_test_setup(self):
```

Comment:
If this is a base method that subclasses are permitted/expected to override, can we remove the `_` prefix? (I know, we have `_get_queryset()` overridden in plenty of places, but that one bugs me too...!)